src/utils/components/TabModal/TabModal.tsx

```diff
@@ -56,35 +57,39 @@ const TabModal: TabModalFC = React.memo(
     positionTop = true,
     submitBtnVariant,
     titleIconVariant,
+    modalError,
   }) => {
     const { t } = useKubevirtTranslation();
 
     const [isSubmitting, setIsSubmitting] = React.useState(false);
-    const [error, setError] = React.useState(undefined);
+    const [apiError, setApiError] = React.useState(undefined);
```

Comment:
can u please remove all * as React form this file?
Also can u set a type for this state?

Reply:
unfortunately, there is no type available for the error in `useK8sWatchResource` or any other sdk method.